stacklet/client/sinistral/commands/run.py

```diff
@@ -71,22 +70,32 @@ def run(ctx, project, dryrun, *args, **kwargs):
     sinistral = sinistral_client()
 
     projects_client = sinistral.client("projects")
-    policy_collections_client = sinistral.client("policy-collections")
 
-    results = []
-    project_data = projects_client.get(name=SinistralFormat.project)
+    try:
+        try:
+            policies = projects_client.get_policies(name=project)
+        except Exception as e:
+            # try to auto-create project if not found
+            if "not found" in str(e):
+                # TODO: should we support default user groups as well?
+                projects_client.create(name=project, groups={"read": []})
```

Comment:
I'm not sure why we're doing this. we want to support the use case, but its not good default behavior, a typo results in a new project?

Reply:
It was for https://github.com/stacklet/sinistral/issues/311 item number 2 in Implementation Notes. I didn't catch the "only when the org creds are used" bit, but could add that. My understanding was that it was to streamline the CI process and remove the whole `sinistral projects get || sinistral projects create` boilerplate.

Reply:
I went ahead and made it only apply when using Org Credentials, since that's the only case where this makes sense anyway.